### ISSUE TITLE
Add Windows Support

### ISSUE THREAD

**Issue by dblock** [opensearch-project/index-management-dashboards-plugin#114]
Coming from https://github.com/opensearch-project/opensearch-plugins/issues/95, add Windows support.

- [ ] Passing CI on Windows
- [ ] Documentation

**Comment by bbarani** [comment]
@praveensameneni Can you please update the status of this issue?

**Comment by dblock** [comment]
I don't believe integration tests are running, and integration tests are very much part of CI. Reopening. Please close when this and any other gaps between Linux/Windows CI/CD are covered.

**Comment by bowenlan-amzn** [comment]
cypress test support for windows is pending for 1.3.7
refer to: https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/370

**Comment by peterzhuamazon** [comment]
Hi, just want to make sure this issue is closed, if all the tasks are completed.

Thanks.

**Comment by CEHENKLE** [comment]
@praveensameneni Can you please address @dblock 's questions?

**Comment by CEHENKLE** [comment]
@praveensameneni @opensearch-project/index-management Can you address @dblock questions?

**Comment by praveensameneni** [comment]
The tests are running  as verified in the logs - 
transforms_spec.js.mp4
templates.js.mp4
split_index.js.mp4
snapshots_spec.js.mp4
rollups_spec.js.mp4
reindex_spec.js.mp4
policies_spec.js.mp4
managed_indices_spec.js.mp4
indices_spec.js.mp4
create_index.js.mp4
aliases.js.mp4

However, per the comment from [issue # 370](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/370)  , we will have to wait until it's resolved in 2.6 . Tracking it for 2.6
